de.klayout.KLayout.yaml

```
@@ -0,0 +1,41 @@
+app-id: de.klayout.KLayout
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: klayout
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --filesystem=host
+- --device=dri
+modules:
+- name: gdbm
+  config-opts:
+  - --enable-libgdbm-compat
+  sources:
+  - type: archive
+    url: ftp://ftp.gnu.org/gnu/gdbm/gdbm-1.18.1.tar.gz
+    sha256: 86e613527e5dba544e73208f42b78b7c022d4fa5a6d5498bf18c8d6f745b91dc
+- name: ruby
+  config-opts:
+  - --enable-shared
+  sources:
+  - type: archive
+    url: https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.gz
+    sha256: d418483bdd0000576c1370571121a6eb24582116db0b7bb2005e90e250eae418
+- name: klayout
+  buildsystem: simple
+  build-commands:
+  - bash ./build.sh -bin ${FLATPAK_DEST}/bin -j$FLATPAK_BUILDER_N_JOBS
+  post-install:
+  - install -Dm644 de.klayout.KLayout.metainfo.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.metainfo.xml
```

```suggestion
  - install -Dm644 de.klayout.KLayout.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```
appdata is the old install directory